pixart.js

```diff
@@ -1 +1,31 @@
+//* When I click the "Set Color" button, it should change the color of the "brush" box to the color I specify in the input field. (**Hint:** You will need to use `event.preventDefault()` somewhere in your code.)
+function colorChange(){
+  var color = window.getComputedStyle(document.getElementById("SetColor"))["background-color"];
```

Comment:
Why do you want to get the background color of the box currently? This will just give you a string like "white".

 Instead, I think you want the text typed into the text box, and use that to **set** (not get) the color of the box.